### ISSUE TITLE
Train Seq2Seq Model in hw2_1

### ISSUE THREAD

**Issue by Johnzu-2019** [JasonYao81000/MLDS2018SPRING#9]
I would like to know why the data from the test set is used during the training phase.
Shouldn't training sets be used?
